### PR TITLE
[nri-bundle] Adding chart for operator

### PR DESCRIPTION
**Pull request by paologallinaharbur**
#### Is this a new chart
yes

#### What this PR does / why we need it:
In order to add newrelic-infra-operator chart to the bundle

#### Which issue this PR fixes
  - fixes https://github.com/newrelic/helm-charts/issues/333
  - fixes https://github.com/newrelic/helm-charts/issues/360

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Review by invidian — APPROVED** [review]

File: charts/nri-bundle/README.md [hunk 1]
```diff
@@ -6,23 +6,24 @@ This chart bundles multiple New Relic products helm-charts.
...
+| `global.licenseKey`               | The [license key][1] for your New Relic Account. This will be preferred configuration option if both `licenseKey` and `customSecret` are specified.  |         |
+| `global.insightsKey`              | The [license key][1] for your New Relic Account. This will be preferred configuration option if both `insightsKey` and `customSecret` are specified. |         |
+| `global.customSecretName`         | Name of the Secret object where the license key is stored                                                                                            |         |
+| `global.customSecretLicenseKey`   | Key in the Secret object where the license key is stored.                                                                                            |         |
+| `global.customSecretInsightsKey`  | Key in the Secret object where the insights key is stored.                                                                                           |         |
+| `global.fargate`                  | Must be set to `true` when deploying in an EKS Fargate environment.                                                                                  |         |
```
[the remark this answers is not in the document]
Comment (paologallinaharbur) [reply charts/nri-bundle/README.md:17]: yes, it should have been false already, I fix that!

**Review by invidian — APPROVED** [review]
LGTM, awesome work @paologallinaharbur! :tada:

File: charts/nri-bundle/README.md [hunk 2]
```diff
@@ -6,23 +6,24 @@ This chart bundles multiple New Relic products helm-charts.
...
+| `ksm.enabled`                     | Install the [`kube-state-metrics` chart from the stable helm charts repository][2]                                                                   | false   |
+| `kubeEvents.enabled`              | Install the [`nri-kube-events` chart][6]                                                                                                             | false   |
+| `logging.enabled`                 | Install the [`newrelic-logging` chart][7]                                                                                                            | false   |
+| `newrelic-pixie.enabled`          | Install the [`newrelic-pixie`][8] chart                                                                                                              | false   |
+| `pixie-chart.enabled`             | Install the [`pixie-chart` chart][9]                                                                                                                 | false   |
+| `newrelic-infra-operator.enabled` | Install the [`newrelic-infra-operator` chart][10]                                                                                                    | false   |
```
Comment (roobre) [inline charts/nri-bundle/README.md:26]:
```suggestion
| `newrelic-infra-operator.enabled` | Install the [`newrelic-infra-operator` chart][10] (Beta)                                                                                              | false   |
```

**Review by roobre — APPROVED** [review]
LGTM, great work!

**Pull request merged**